src/ankh/cmd/ankh/main.go

```diff
@@ -261,3 +272,52 @@ func check(err error) {
 		os.Exit(1)
 	}
 }
+
+func runScripts(ctx *ankh.ExecutionContext, scripts []struct { Path string }) {
+	for _, script := range scripts {
+		path := script.Path
+		if path == "" {
+			log.Warnf("Missing path in script %s", script)
+			break
+		}
+		info, err := os.Stat(path)
+		if os.IsNotExist(err) {
+			log.Fatalf("Script not found: %s", path)
+			os.Exit(1)
+		}
+		// get the mode bits and check that the script is executable by someone
```

Comment:
we can use https://linux.die.net/man/2/access (there is likely a golang variant) to check that the script is executable by the running process, instead of just someone

Reply:
Sure, I'll do a shell exec with `access` to check that then

Reply:
Oh nvm, this exists: https://golang.org/pkg/syscall/#Access